### ISSUE TITLE
Genome version: 20201 is INCOMPATIBLE with running STAR version: 2.7.1a

### ISSUE THREAD

**Issue by sjackman** [alexdobin/STAR#747]
```console
$ STAR --version
2.7.1a
$ STAR --runThreadN=1 --genomeDir=grch38/star --readFilesIn=diff.fq --outFileNamePrefix=diff.star/Aligned.out.sam
Sep 26 13:20:57 ..... started STAR run
Sep 26 13:20:57 ..... loading genome

EXITING because of FATAL ERROR: Genome version: 20201 is INCOMPATIBLE with running STAR version: 2.7.1a
SOLUTION: please re-generate genome from scratch with running version of STAR, or with version: 2.7.1a

Sep 26 13:20:57 ...... FATAL ERROR, exiting
```

I believe the index version 20201 was created with STAR 2.7.1a, and I'm running STAR version 2.7.1a. What's gone wrong here?

**Comment by sjackman** [comment]
After some inspection, it looks like this genome index was built with STAR 2.5.1b. Could the error message be improved?

**Comment by alexdobin** [comment]
Hi Shaun,

I think the 20201 version is confusing, but this is already recorded in the genome index that was generated with older STAR version. Starting from 2.7.x, it will actually list the version it was generated with.

Cheers
Alex

**Comment by sjackman** [comment]
Hi, Alex. I've sorted out this issue on my end, so feel free to close this issue.

**Comment by lgoehring** [comment]
Hi @sjackman, would you be able to share how you sorted this out? @alexdobin

**Comment by alexdobin** [comment]
Hi @lgoehring 

this was solved in the 2.7.4 release.
The easiest way to generate a compatible genome is to do it with the same version of STAR you are using for mapping.

Cheers
Alex

**Comment by bioinfotec** [comment]
> Hi, Alex. I've sorted out this issue on my end, so feel free to close this issue.

How do you solve this problem? I encountered the same problem recently.
